graphstorm-processing/docker/build_gsprocessing_image.sh

```diff
@@ -120,12 +135,22 @@ fi
 # Copy Docker entry point to build folder
 cp ${GSP_HOME}/docker-entry.sh "${BUILD_DIR}/docker/code/"
 
-DOCKER_FULLNAME="${IMAGE_NAME}:${VERSION}"
+# Export Poetry requirements to requirements.txt file
+poetry export -f requirements.txt --output "${BUILD_DIR}/docker/requirements.txt"
+
+# Set image name
+DOCKER_FULLNAME="${IMAGE_NAME}-${EXEC_ENV}:${VERSION}"
 
 # Login to ECR to be able to pull source SageMaker image
-aws ecr get-login-password --region us-west-2 \
-    | docker login --username AWS --password-stdin 153931337802.dkr.ecr.us-west-2.amazonaws.com
+if [[ ${EXEC_ENV} == "sagemaker" ]]; then
+    aws ecr get-login-password --region us-west-2 \
+        | docker login --username AWS --password-stdin 153931337802.dkr.ecr.us-west-2.amazonaws.com
+else
+    aws ecr-public get-login-password --region us-east-1 | docker login --username AWS --password-stdin public.ecr.aws
```

Comment:
A note here @classicsong `aws ecr-public` and `aws emr-serverless` commands are only available for AWS CLI v2. The command will error out here if AWS v1 is being used, just wondering if we should fail eariler and inform the user.

Reply:
I would prefer fail eariler. Can we check the CLI version and fail when it is v1?